src/utils/input-validators.js

```diff
@@ -5,6 +5,21 @@ function isValidPassword({ password, minLength = 8 }) {
   return sanitizedPassword.length >= minLength && minOneLetterAndOneDigitPattern.test(sanitizedPassword);
 }
 
+function isValidTag(allTags, tag) {
+  const trimmedTag = (tag || '').trim();
+
+  if (trimmedTag.length < 3 || trimmedTag.length > 25 || trimmedTag.includes(' ')) {
```

Comment:
Max Characters should be 30 according to this: https://github.com/elmu/elmu-web/issues/184

Also, I would check for whitespace using `/\s/`, not only the space characters, because people will use tabs and all kind of whitespace chars they copy out of their word documents or the internet :-)